exe/RadicleExe.hs

```diff
@@ -76,9 +88,20 @@ opts = Opts
           <> "where $DIR is $XDG_DATA_HOME (%APPDATA% on Windows "
           <> "if that is set, or else ~/.local/share."
            )
-        ))
+       ))
+    <*> many (strArgument mempty)
+
+argsPrimFn :: forall m . Monad m => [Text] -> PrimFns m
+argsPrimFn args = fromList $ allDocs
+    [ ( "get-args!"
+      , "Returns the list of the command-line arguments the script was called with"
+      , \case
+          [] -> pure . Vec $ fromList (String <$> args)
+          xs -> throwErrorHere $ WrongNumberOfArgs "get-args!" 0 (length xs)
+      )
+    ]
```

Comment:
Can we move this to where all the other primitive functions live? Ideally this should be implemented with capabilities. But we could also have `replBindings` take an argument.